src/reducers/ui.js

```diff
@@ -77,9 +83,15 @@ export default function ui(stateIn, action) {
     case 'EDITOR_FOCUSED_REQUESTED_LINE':
       return state.setIn(['editors', 'requestedFocusedLine'], null);
 
-    case 'EDITORS_UPDATE_VERTICAL_FLEX':
-      return state.updateIn(['editors', 'verticalFlex'], (prevFlex) => {
-        const newFlex = updateVerticalFlex(action.payload);
+    case 'WORKSPACE_UPDATE_COLUMN_FLEX':
```

Comment:
Riffing on the naming question above, I’d think about naming these actions something that is more focused on user intent rather than what specifically we’re going to do—e.g. maybe `MOVE_COLUMN_DIVIDER`/`MOVE_ROW_DIVIDER`??